core/src/main/resources/hudson/triggers/Messages.properties

```
@@ -35,3 +35,4 @@ TimerTrigger.the_specified_cron_tab_is_rare_or_impossible=This schedule will mat
   never (e.g. June 31), so this job may be triggered very rarely, if at all.
 Trigger.init=Initializing timer for triggers
 SCMTrigger.AdministrativeMonitorImpl.DisplayName=Too Many SCM Polling Threads
+SlowTriggerAdminMonitor.DisplayName=Cron triggers monitor
```

```suggestion
SlowTriggerAdminMonitor.DisplayName=Cron Triggers Monitor
```

All others are title-cased too.